source/org/zfin/db/postGmakePostloaddb/1125/ZFIN-7584.sql

```diff
@@ -0,0 +1,17 @@
+--liquibase formatted sql
+--changeset cmpich:ZFIN-7584
+
+
+update feature_dna_mutation_detail
+set fdmd_dna_mutation_term_zdb_id = vocab.mdcv_term_zdb_id
+from mutation_detail_controlled_vocabulary vocab,
+     ensdarg_temp
+where fdmd_feature_zdb_id = et_id
+  AND mdcv_term_display_name = et_reference || '>' || et_variant;
+
+select * from feature_dna_mutation_detail,
```

Comment:
What does a select statement do in a liquibase sql file?

Reply:
I had thought the output of the select query would be displayed somewhere so I know how many changes are done. But yes, not really necessary.